src/templates/hi/2020/base.html

```diff
@@ -0,0 +1,10 @@
+{% extends "%s/2019/base.html" % lang %}
+
+{% block methodology_stat_1 %}7.5M{% endblock %}
+{% block methodology_stat_2 %}31.3 TB{% endblock %}
+{% block methodology_description %}
+जब तक अन्यथा नोट न किया जाए, Web Almanac के सभी 22 अध्यायों में मौजूद मेट्रिक्स HTTP Archive के स्रोतों से लिये जाते हैं। HTTP Archive एक समुदाय द्वारा संचालित परियोजना है, जो 2010 के बाद से वेब कैसे बनाया जा रहा है इस पर निगाह रखें हुए हैं। WebPageTest और Lighthouse का अति उत्कृष्ट उपयोग करते हुए, 7.5 मिलियन से अधिक वेबसाइटों के बारे में मेटाडेटा को मासिक रूप से परीक्षण किया जाता है और विश्लेषण के लिए एक सार्वजनिक BigQuery डेटाबेस में शामिल किया जाता है। अगस्त 2020 डेटासेट का उपयोग Web Almanac के मैट्रिक्स के आधार के रूप में किया गया था। अधिक जानकारी के लिए, कार्यप्रणाली पृष्ठ देखें।
+{% endblock %}
+
+{% block foreword %}
+{% endblock %}
```

Comment:
Nit but can we have a newline at the end of files please.

```suggestion
{% endblock %}

```

Reply:
Sure